core/capabilities/integration_tests/framework/capabilities_registry.go

```diff
@@ -0,0 +1,136 @@
+package framework
+
+import (
+	"context"
+
+	"github.com/ethereum/go-ethereum"
+	"github.com/ethereum/go-ethereum/accounts/abi/bind"
+	"github.com/ethereum/go-ethereum/common"
+	"google.golang.org/protobuf/proto"
+
+	"github.com/smartcontractkit/chainlink-common/pkg/capabilities/pb"
+	"github.com/smartcontractkit/chainlink-common/pkg/values"
+	kcr "github.com/smartcontractkit/chainlink/v2/core/gethwrappers/keystone/generated/capabilities_registry"
+
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+const (
+	CapabilityTypeTrigger   = 0
```

Comment:
@ettec I'd consider moving this elsewhere so that others can import: I think we have a few of these kicking around now.

Reply:
as these are the capability types as known in the registry contract, and therefore also the syncer, I've put an enum in the syncer along with a replacement to the existing switch that converts the contract capability type to the common capability type